wiki/Beatmap/Pattern/Mania/Jacks/en.md

```diff
@@ -0,0 +1,31 @@
+---
+stub: true
+tags:
+  - jacks
+---
+
+# Jacks
+
+Within specific pattern context, **jacks** are typically used to refer to 3 or more consecutive notes within the same column. They are referred to as **jacks** since the motion required to play them mimics the motion of a jackhammer.
+
+![](img/jack.png "This image displays a jack within it's traditional usage of the word.")
+
+Commonly, the term **jack** can also refer to the general idea of having consecutive notes in the same column. Any of the following specifications can be referred to as just **jacks**.
+
+## Minijack
+
+A **minijack** is a type of **jack** which only features two notes and is the least demanding version of a **jack**.
+
+![](img/minijack.png)
+
+## Longjack
+
+A **longjack** is a more strenuous type of **jack** which typically uses 4 or more consecutive notes. This term may also be used when the same consecutive jumps or hands are used.
```

Comment:
I don't think there is much of a difference between a longjack and an anchor right now, so maybe add a diferenciation here.

Reply:
I think maybe mentioning that longjacks typically refer to isolated to an isolated pattern might help. 
```suggestion
A **longjack** is a more strenuous type of **jack** which uses 4 or more consecutive notes, typically isolated from any other pattern. This term may also be used when the same consecutive jumps or hands are used.
```
What do you think?
Though do keep in mind, for anchors I specified that they are specifically a byproduct of other patterns, and as such require other patterns to exist.